FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
If it is all in a header file then there is no need to explicitly add the package to the BuildFIle.xml. I understand your reluctance but I also do not want FWCore/Concurrency to hold files which have nothing to do with concurrency.

@smuzaffar what is your opinion on a new package Utilities/Xerces?